### PR TITLE
Add search API

### PR DESCRIPTION
**Pull request by vitslepukhin**

### CODE REVIEW

File: src/app/ng-swagger-gen.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "includeTags": ["search"],
+  "prefix": "Api",
+  "ignoreUnusedModels": true,
+  "minParamsForContainer": 2,
+  "sortParams": "desc",
+  "defaultTag": "Api",
+  "removeStaleFiles": true,
+  "modelIndex": true,
+  "serviceIndex": true,
+  "apiModule": true,
```
Comment (enkryptor) [inline src/app/ng-swagger-gen.json:14]: maybe should keep it `false`

**Review by ennaque — CHANGES_REQUESTED** [review]
need to fix code style

**Review by enkryptor — APPROVED** [review]

**Review by ennaque — APPROVED** [review]

File: src/app/search/search-page.component.scss [hunk 2]
```diff
@@ -24,3 +24,8 @@ ion-searchbar {
   font-size: 24px;
   font-weight: bold;
 }
+
+.search-result {
+  display: block;
```
Comment (enkryptor) [inline src/app/search/search-page.component.scss:29]: let's move this to the `:host`: block in the component scss

File: src/app/search/components/search-result/search-result.component.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import {Component, Input} from '@angular/core';
 import {HelperService} from '@app/core/services/helper.service';
 import {PhotoSanitizerService} from '@app/core/services/photo-sanitizer.service';
-import {SearchResultsInterface} from '@app/search/interfaces/search-results-interface';
 import {Service} from '@app/service/models/service';
+import {Search, ServiceList} from '../../../api/models';
```
Comment (enkryptor) [inline src/app/search/components/search-result/search-result.component.ts:5]: let's use absolute path here

**Pull request merged**